src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -2479,6 +2506,22 @@ export class StudyViewPageStore {
         default: []
     });
 
+    readonly fusionGeneData = remoteData<FusionGenesData>({
+        await: () => [this.mutationProfiles],
+        invoke: async () => {
+            if (!_.isEmpty(this.mutationProfiles.result!)) {
```

Comment:
not too important, but you might not have to use ! after result here.  one would think that isEmpty typing would accept undefined since it's meant to test for undefined (among other things).

Reply:
I see. thanks for pointing this out. Will remove !.

Reply:
! removed.